### PR TITLE
Extended React motion Menu

### PR DESCRIPTION
**Pull request by hasibsahibzada**
We liked the React-motion-menu, because it is very easy to use and understandable. 
To give user more control over the menu, we have introduced some extra props for the MotionMenu component. 

**New Props** 
1. bumpy: (bool): This prop controls if the menu Items should bump or open smoothly.
2. openSpeed: (number): (miliseconds) This controls the speed of the menu while opening. 
3. reverse: (bool) : This controls if the menu should open in reverse order.

### CODE REVIEW

File: docs/src/example.js [hunk 1]
```diff
@@ -3,10 +3,14 @@ import MotionMenu from '../../src';
 
 export default () => (
   <MotionMenu
-    type="circle"
+    type="vertical"
```
Comment (bokuweb) [inline docs/src/example.js]: Please set `circle` as type 🙏

File: docs/src/example.js [hunk 2]
```diff
@@ -3,10 +3,14 @@ import MotionMenu from '../../src';
...
     margin={120}
     y={0}
+    bumpy={false}
```
Comment (bokuweb) [inline docs/src/example.js]: Please set true fore example.

File: docs/src/example.js [hunk 3]
```diff
@@ -3,10 +3,14 @@ import MotionMenu from '../../src';
...
     x={0}
+    openSpeed={10}
```
Comment (bokuweb) [inline docs/src/example.js]: Please Set {60} for example

**Comment by bokuweb** [comment]
@hasibsahibzada thanks!! it looks great to me 👀 
please fix items pointes out , and add new props description to README property.

**Comment by hasibsahibzada** [comment]
Thank you very much for your reviews and feedbacks.

**Comment by bokuweb** [comment]
@hasibsahibzada thanks!!

**Comment by bokuweb** [comment]
published as v0.4.0 🎉

**Pull request merged**